bin/merge_results.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
```

Comment:
add the missing Apache licence code snippet

Reply:
Good catch ;D